lldb/tools/lldb-dap/package.json

```
@@ -78,6 +78,15 @@
           "scope": "resource",
           "type": "string",
           "description": "The log path for lldb-dap (if any)"
+        },
+		"lldb-dap.environment": {
```

Use whitespaces instead of tabs for indentation

```suggestion
        },
        "lldb-dap.environment": {
```